content/templates/profile.md

```diff
@@ -1,13 +1,20 @@
 ---
-created: 2023-11-17
-title: 
+created: 2023-11-30
+contributors: 
+title: title of the wiki entry, e.g. name of organisation, person, or concept
 url: 
-city: 
-country: 
+locations: 
+started: 
+size:
+  - 1-3
```

Comment:
So for now- we have to choose which size it is?

Reply:
@matthewmccarthy11 -- yes, like we were doing from the beginning originally in the google sheet